src/ASM/LR/ASMu3Dmx.h

```diff
@@ -17,6 +17,8 @@
 #include "ASMu3D.h"
 #include "ASMmxBase.h"
 
+class Fields;
+
```

Comment:
Not needed, since done in ASMbase.h which is included

Reply:
removed.